test/e2e/reconciler/namespace/controller_test.go

```diff
@@ -220,21 +219,6 @@ func TestNamespaceScheduler(t *testing.T) {
 
 				err = server.expect(namespace, unscheduledMatcher(nscontroller.NamespaceReasonSchedulingDisabled))
 				require.NoError(t, err, "did not see namespace marked with scheduling disabled")
-
-				t.Log("Assign a cluster to the namespace manually")
```

Comment:
Was this purposefully removed?

Reply:
Yes, is intentional. The removed part of this test was checking that a namespace for which scheduling is disabled and a cluster is assigned has a condition of scheduled=true (implying automatic scheduling). As per the unit tests, the desired condition should be scheduled=false with reason 'SchedulingDisabled' (implying assignment is handled manually). Since we already have unit test coverage for this I chose to remove the broken test rather than rewrite.

Reply:
Or should any assigned cluster - regardless of whether automatically assigned - imply a condition of scheduled=true?  Definitely worth some thought as to how we want this to behave from a UX perspective.